### PR TITLE
fix/PLF-8141:No timeout specified on the SMTP Transport for email not…

### PR DESCRIPTION
**Pull request by lbengayed**
…ification

### CODE REVIEW

File: extension/config/src/main/resources/conf/platform/exo-sample.properties [hunk 1]
```diff
@@ -98,6 +98,15 @@
 # This class will be used to create SMTP sockets.
 # Sample: exo.email.smtp.socketFactory.class=javax.net.ssl.SSLSocketFactory
 #exo.email.smtp.socketFactory.class=
+# Specify the timeout Socket connection.
```
Comment (ahamdi) [inline extension/config/src/main/resources/conf/platform/exo-sample.properties]: Could you provide description for those parameters as you did in configuration.properties

File: extension/config/src/main/resources/conf/platform/exo-sample.properties [hunk 2]
```diff
@@ -98,6 +98,15 @@
...
+# Sample: email.smtp.connectiontimeout=0
+#email.smtp.connectiontimeout=
+# Sample: email.smtp.timeout=0
+#email.smtp.timeout=
+# Sample: email.smtp.writetimeout=0
+#email.smtp.writetimeout
+# Sample: email.smtp.debug:false
```
Comment (ahamdi) [inline extension/config/src/main/resources/conf/platform/exo-sample.properties]: it should be : email.smtp.debug=false

Comment (lbengayed) [reply extension/config/src/main/resources/conf/platform/exo-sample.properties]: ok

File: extension/config/src/main/resources/conf/platform/configuration.properties [hunk 3]
```diff
@@ -60,6 +60,14 @@ gatein.email.smtp.starttls.enable=${exo.email.smtp.starttls.enable:false}
 gatein.email.smtp.auth=${exo.email.smtp.auth:false}
 # Username to send for authentication.
 gatein.email.smtp.username=${exo.email.smtp.username:}
+# Socket connection timeout.
+gatein.email.smtp.connectiontimeout=${email.smtp.connectiontimeout:0}
```
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: All properties must be prefixed by "exo.", so for exampe here it should be "exo.email.smtp.connectiontimeout"

**Pull request merged**